src/TikaOnDotNet.Tests/text_extraction.cs

```diff
@@ -48,7 +48,7 @@ public void should_extract_mp4()
 		public void should_be_able_to_delete_the_mp4_after_extraction()
 		{
 			var original = new FileInfo(Path.Combine(Directory.GetCurrentDirectory(), @"files\badgers.mp4"));
-			var mp4 = original.CopyTo(Path.Combine(Directory.GetCurrentDirectory(), @"files\badgers.bak.mp4"));
+			var mp4 = original.CopyTo(Path.Combine(Directory.GetCurrentDirectory(), @"files\badgers.bak.mp4"), true);

```

Comment:
I had to allow overwriting here, since the test failed on this line if it had already failed on a previous run.